randovania/data/json_data/prime3.txt

```diff
@@ -1181,6 +1348,11 @@ Asset id: 7492844587871276588
   > Dock to Hidden Court Hall
       Suit Type ≥ 4
 
+> Event - Pickup Golem Head; Heals? False
+  * Event Fuel Gel Pool Golem Head
+  > Dock to Hidden Court Hall
+      Trivial
+
 ----------------
 Jungle Generator
 Asset id: 6613311822671481119
```

Comment:
Room Center should connect to the shield event, with requirements: Ship Missile and After North Jungle Court Turret